packages/oas-normalize/tsup.config.ts

```diff
@@ -1,11 +1,9 @@
-import type { Options } from 'tsup';
```

Comment:
this type can be inferred from the `defineConfig` helper so it's not adding anything here